lib/rubocop/cop/rspec/file_path.rb

```diff
@@ -103,7 +103,9 @@ def ignore_methods?
         end
 
         def filename_ends_with?(glob)
-          File.fnmatch?("*#{glob}", processed_source.buffer.name)
+          filename =
+            RuboCop::PathUtil.relative_path(processed_source.buffer.name)
```

Comment:
It's possible to call `rubocop` from a subdirectory, e.g. `/home/foo/spec/models/`, in this case relative path will be `./bar_spec.rb`, this shouldn't cause any thouble, should it?

Reply:
I think that it depends. If subject is like 'Foo::Bar', it wouldn't detect. I think that most people run from project root

Reply:
Seems this way it doesn't work if I run rubocop and provide a path to check, e.g.
`rubocop --require rubocop-rspec ~/my_project/specs`

Reply:
@eitoball Can you please check this edge case?

Reply:
Oh, I didn't notice. Will do over this weekend. Thank you.

Reply:
A friendly reminder. This is ready to be merged apart from this edge case.

Reply:
Hi guys,
i think i had this edge case. I tried to create a pull request for it:

https://github.com/rubocop-hq/rubocop-rspec/pull/917